bones/bone.py

```diff
@@ -685,32 +685,44 @@ def mergeFrom(self, valuesCache, boneName, otherSkel):
 			return
 		valuesCache[boneName] = copy.deepcopy(otherSkel.valuesCache.get(boneName, None))
 
-	def setBoneValue(self, skel, boneName, value, append, *args, **kwargs):
+	def setBoneValue(self, skel, boneName: str, value: Any, append: bool, language: Union[None, str] = None) -> bool:
 		"""
 			Set our value to 'value'.
-			Santy-Checks are performed; if the value is invalid, we flip our value back to its original
-			(default) value and return false.
+			Santy-Checks are performed; if the value is invalid, no modification will happen.
 
-			:param valuesCache: Dictionary with the current values from the skeleton we belong to
-			:type valuesCache: dict
+			:param skel: Dictionary with the current values from the skeleton we belong to
+			:type skel: SkeltonInstance
 			:param boneName: The Bone which should be modified
 			:type boneName: str
 			:param value: The value that should be assigned. It's type depends on the type of that bone
 			:type boneName: object
 			:param append: If true, the given value is appended to the values of that bone instead of
 				replacing it. Only supported on bones with multiple=True
 			:type append: bool
+			:param language: Set/append which language
+			:type language: str or None
 			:return: Wherever that operation succeeded or not.
 			:rtype: bool
 
 		"""
```

Comment:
i'am not really satisfied with this types here.
The type annotations and docstring-types are sometimes different (must boneName be a str or object) or they're missing (Skel[e]tonInstance).
I think we don't need them redudant and should remove one kind of them.
I suggest to eliminate the types in the docstring and use only the type annotations in the signature.